docs/src/network.md

```diff
@@ -3,16 +3,13 @@ id: network
 title: "Network"
 ---
 
-Playwright provides APIs to **monitor** and **modify** network traffic, both HTTP and HTTPS. Any requests that page
-does, including [XHRs](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) and
+Playwright provides APIs to **monitor** and **modify** network traffic, both HTTP and HTTPS. Any requests that a page does, including [XHRs](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) and
 [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) requests, can be tracked, modified and handled.
 
-<!-- TOC -->
-
-<br/>
-
 ## HTTP Authentication
 
+Check HTTP Authentication with [`method: Browser.newContext`].
```

Comment:
I'm not sure if "Check" is the right term here since playwright automatically performs authentication with the specified credentials if the web site requests it, but it doesn't actually check.

Reply:
so change check to "perform"

Reply:
Sounds good!